.travis.yml

```diff
@@ -23,8 +23,10 @@ before_install:
   - pip install -U pip
   - pip install codecov
 install:
+  - pip install -r requirements-dev.txt
   - pip install  ".[test]" .
 script:
+  - pre-commit run -a
```

Comment:
With this, the flake8 command on line 34 can be removed

Reply:
done